extractors/betydb/terra_betydb.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+from pyclowder.utils import CheckMessage
+from pyclowder.files import download_metadata, upload_metadata
+from terrautils.extractors import TerrarefExtractor, is_latest_file, load_json_file, \
+    build_metadata, build_dataset_hierarchy
+from terrautils.betydb import add_arguments, get_sites, get_sites_by_latlon, submit_traits, \
+    get_site_boundaries
+from terrautils.metadata import get_extractor_metadata, get_terraref_metadata
+
+
+class BetyDBUploader(TerrarefExtractor):
+    def __init__(self):
+        super(BetyDBUploader, self).__init__()
+
+        # parse command line and load default logging configuration
+        self.setup(sensor='stereoTop_canopyCover')
```

Comment:
I don't understand why this is set to canopycover, or why we need a sensor specified here at all?  The bety post knows what the traits are based on the CSV contents.

Reply:
terrautils TerraRefExtractor currently requires at minimum an existing sensor name. it isn't used anywhere here and doesn't affect anything. we can refactor it at some point to accept no sensor but right now it will throw error if there isn't a valid name given here.